src/randomizer.py

```diff
@@ -2080,6 +2080,102 @@ def visit_Decl(self, decl: ca.Decl) -> None:
     ast_util.insert_decl(fn, var, type, random)
 
 
+def perm_inline_get_structmember(
+    fn: ca.FuncDef, ast: ca.FileAST, indices: Indices, region: Region, random: Random
+) -> None:
+    """Creates an inline for accessing a struct member."""
+
+    typemap = build_typemap(ast)
+
+    ext_fns: List[str] = []
+    cands: List[ca.StructRef] = []
+
+    for item in ast.ext:
+        if (
+            isinstance(item, ca.Decl)
+            and isinstance(item.type, ca.FuncDecl)
```

Comment:
```suggestion
```
and rename ext_fns -> used_names and make it a set

Reply:
check